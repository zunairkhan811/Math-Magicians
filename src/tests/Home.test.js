import React from 'react';
import { render } from '@testing-library/react';
import Home from '../components/Home';

test('renders Home component correctly', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
