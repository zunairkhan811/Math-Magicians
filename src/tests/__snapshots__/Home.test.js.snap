// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component correctly 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <h1>
      Welcome to our Page!
    </h1>
    <p>
      I am Excited that you are Here. Math Magicians Website has been made for user to  Do Calculation online. Feel free to visit our Calculator App by clicking on Calculator Tab and do the calculation as many as you want. If you wanna see some quotes as well, you can see the quotes by clicking on the Quotes Tab on Top right of the page.
    </p>
  </div>
</DocumentFragment>
`;
